server/store/datastore/migration/019_gated_to_require_approval.go

```diff
@@ -26,10 +26,8 @@ var gatedToRequireApproval = xormigrate.Migration{
 	ID: "gated-to-require-approval",
```

Comment:
Would suggest to copy the exact migration from 2.8 here. It is currently missing the removal from the `gated` column. https://github.com/woodpecker-ci/woodpecker/blob/985a5ae73bf31e93e58f4c123a974b1de5bce556/server/store/datastore/migration/034_gated_to_require_approval.go